test/unit/pdf_find_controller_spec.js

```diff
@@ -554,4 +555,48 @@ describe("pdf_find_controller", function () {
       pageMatchesLength: [[23]],
     });
   });
+
+  it("performs a search in a text containing some -\\n", async function () {
```

Comment:
Maybe `"performs a search in a text containing diacritics before -\\n"` to more clearly describe the problem you're fixing here (based on the PR description).